asset/gulpfile.js

```diff
@@ -65,13 +65,14 @@ gulp.task('scripts', function () {
                 'node_modules/prismjs/plugins/line-numbers/prism-line-numbers.js',
                 'node_modules/jquery/dist/jquery.js',
                 'node_modules/bootstrap-sass/assets/javascripts/bootstrap.js',
+                'node_modules/anchor-js/anchor.js',
                 'node_modules/choices.js/public/assets/scripts/choices.js',
                 'js/component-list.js',
                 'js/base.js'
             ]
         )
     )
-        .pipe(babel({presets: ['@babel/env']}))
+        .pipe(babel({presets: ['@babel/env'], sourceType: 'unambiguous'}))
```

Comment:
Out of curiousity: how does this fix the issues I was observing in previous builds?

Essentially, the following construct was problematic when I was running builds previously:

```javascript
(function (root, factory) {
  /* ... */
    // Browser globals (root is window)
    root.AnchorJS = factory();
 /* ... */
}(this, function () { /* .. */ }
```

What I was getting was an error indicating that the JS engine could not set the property `AnchorJs` on an unknown type. Does the flag above solve that problem during build somehow?

Reply:
Yes, the problem with the `TypeError` and on setting `AnchorJS` is solved with this solution.

Reply:
I tested it, and it works wonderfully!

Still want to know _why_ at some point, because I would have never figured that out on my own!

Reply:
I had the same problem in another project and found the hint in my notes.

The import part from the Babel documentation:

> `unambiguous` - Consider the file a "module" if `import/export` statements are present, or else consider it a "script".
> ...
> Setting the correct `sourceType` can be important because having the wrong type can lead to cases where Babel would insert import statements into files that are meant to be CommonJS files. This can be particularly important in projects where compilation of `node_modules` dependencies is being performed, because inserting an `import` statements can cause Webpack and other tooling to see a file as an ES module, breaking what would otherwise be a functional CommonJS file.

https://babeljs.io/docs/en/options#sourcetype